src/Treeview.ts

```diff
@@ -169,6 +170,12 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
             this.tree = Json.parse(this.text);
         }
     }
+    private getContextValue(elementInfo: IElementInfo): string {
+        if (elementInfo.current.level === 1) {
+            return this.getTreeNodeLabel(elementInfo);
```

Comment:
This could be an issue when we localize, would prefer to see it return something that won't get localized:

  private getContextValue(elementInfo: IElementInfo): string {
    if (elementInfo.current.level === 1) {
      const keyNode = this.tree && this.tree.getValueAtCharacterIndex(elementInfo.current.key.start);
      if (keyNode instanceof Json.StringValue) {
        return keyNode.unquotedValue;
      }
    }

    return undefined;
  }

Reply:
Fixed.